workflows4s-bpmn/src/main/scala/workflows4s/bpmn/BPMNConverter.scala

```diff
@@ -1,11 +1,11 @@
 package workflows4s.bpmn
 
-import java.time.Duration
+import cats.implicits.catsSyntaxOptionId
```

Comment:
unused import and there are couple more

Reply:
Good point, I made them fatal errors and fixed all of it.